docs/reference/query-dsl/constant-score-query.asciidoc

```diff
@@ -7,12 +7,16 @@ filter. Maps to Lucene `ConstantScoreQuery`.
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
+  "query": {
```

Comment:
I'm not sure if it is ok to confuse indention? I like two spaces and some of the snippets use that but this uses both two and four spaces now.